ocaml/testsuite/tests/jane-modular-syntax/attribute_locations.ml

```diff
@@ -0,0 +1,70 @@
+(* TEST
+   include ocamlcommon *)
+
+let () = Language_extension.enable Comprehensions ();;
+
+ module Location_map = struct
+   include Map.Make (struct
+     type t = Location.t
+     let compare = compare
+  end)
```

Comment:
What's the reason this isn't just `include Map.Make (module Location)`?

Reply:
This compare is polymorphic compare; there's no `Location.compare`, so `Location` doesn't satisfy the functor parameter module type.

Reply:
:facepalm: of course